README.md

```diff
@@ -78,6 +81,96 @@ You must pass call events handler, registeration event handler (optional) and se
    7. **ToggleHold**: Toggle state hold/un-hold state of a call. This state needs to be maintained by the client
    8. **ToggleMute** Toggle state mute/un-mute state of a call. This state needs to be maintained by the client
 
+
+
+
+
+# For Non-npm integration
+If you are not using a package manager like NPM, you can directly include the bundled JavaScript file that can be generated by webpack.
+
+## Steps
+
+1. Locate the webpack config file in the project directory.
+
+2. Install all the dependencies that are require to run webpack.
+
+3. Run the following command from the root directory of your project: 
+```bash
+npx webpack
+```
+
+4. Now you'll have one folder called  **'target'**  in your root directory.
+
+5. You can find one bundle file called **'crmBundle.js'** in the target folder.
+
+6. Now you can use this **'crmBundle.js'** file by adding it as a script. 
+```js
+<script src="./target/crmBundle.js"></script>
+```
+7. Configure the crmWebSDK object like this:
+```js
+const crmWebSDK = new ExotelCRMWebSDK(accessToken, userId, true);
+```
+
+8. **ExotelCRMWebSDK**
```

Comment:
@Anant-exotel This point about usage of objects in websdk and the ones below are the same for npm or non-npm use-case.
Can we remove this duplication by organising it in a separate common section?
We can organise readme into 2 steps broadly, Installation and Usage.